scss/_settings.scss

```diff
@@ -179,7 +192,7 @@ $fd-padding--ui: $fd-spacing--base * 8 !default;
 $fd-max-width--ui: 1290px !default;
 
 //global widths
-$fd-width--gutter: map-get($fd-spacing, m) !default;
+$fd-width--gutter: var(--fd-width-gutter) !default;
```

Comment:
There are some instances where this value is being used to do some math. See `table`. You need to update all those instances to use `calc` instead of SASS math.

I'm not sure what this gets you though as opposed to just using the CSS var directly.

```
width: calc($fd-width--gutter / 2);
//vs.
width: calc(var(--fd-width-gutter) / 2);
```

I wouldn't mix these up. Use SASS in settings to set the CSS vars and then use the vars in the component CSS. It will be confusing to not know if a SASS var will return a value or a CSS var.